etc/config/haskell.defaults.properties

```diff
@@ -1,8 +1,9 @@
-compilers=/usr/bin/ghc
+compilers=/usr/local/bin/ghc
```

Comment:
Again, don't change defaults. Add your own `haskell.local.properties` where you can override this